packages/mds-agency/tests/integration-tests.spec.ts

```diff
@@ -1274,8 +1277,8 @@ describe('Tests API', () => {
         test.value(deviceA.device_id).is(DEVICE_UUID)
         test.value(deviceA.provider_id).is(TEST1_PROVIDER_ID)
         test.value(deviceA.gps.lat).is(TEST_TELEMETRY.gps.lat)
-        test.value(deviceA.status).is(VEHICLE_STATES.available)
-        test.value(deviceA.prev_event).is(VEHICLE_EVENTS.reservation_cancel)
+        test.value(deviceA.state).is(VEHICLE_STATES.available)
+        test.value(JSON.stringify(deviceA.prev_events)).is(JSON.stringify([VEHICLE_EVENTS.reservation_cancel]))
```

Comment:
Nit, but I don't see why you have to stringify both things.

Reply:
Array equality in javascript is tricky https://masteringjs.io/tutorials/fundamentals/compare-arrays

Reply:
Using JSON.stringify is a bit of a shortcut and has some edge cases that cause problems, but for the sake of this test we don't expect `null | undefined` values in the arrays at all, so should be fine.